config/default.yaml

```diff
@@ -163,3 +163,5 @@ ecosystem:
     store: https://store.screwdriver.cd
     # Badge service (needs to add a status and color)
     badges: https://img.shields.io/badge/build-{{status}}-{{color}}.svg
+    # Default registry to pull build containers from. Uses Docker Hub if nothing/empty string is provided
+    dockerRegistry: ""
```

Comment:
Where is the logic that defaults to dockerhub? Would it make more sense to just have dockerhub in the default.yaml?